### PR TITLE
Issue 19 - Allow traversing to micro-updates

### LINKED ISSUE #19: Allow traversing to micro-updates
We must allow micro-updates behave as independent content to let editors to use them in social media. We can use the time stamp as the URL of the micro-update and add a link to it using the date itself.

Here is an example of a micro-update that must be treated as an individual piece of content:

selection_066

The look and feel must follow Plone standard main template.

### PR DESCRIPTION
**Pull request by rodfersou**
closes #19

### CODE REVIEW

File: src/collective/liveblog/browser/templates/microupdate.pt [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+              <p class="microupdate-title">
+                <strong i18n:translate="">No micro-updates yet for this Liveblog.</strong>
+              </p>
+            </div>
+          </section>
+        </div>
+        <div tal:replace="structure provider:plone.belowcontentbody" />
+          <div class="back"><span i18n:translate="" tal:omit-tag="">Extracted from:</span> <a tal:attributes="href context/absolute_url" tal:content="context/Title" /></div>
+      </article>
+      <script>
```
Comment (hvelarde) [inline src/collective/liveblog/browser/templates/microupdate.pt]: I think this script makes no sense in this context; you can remove it

Comment (rodfersou) [inline src/collective/liveblog/browser/templates/microupdate.pt]: ok, when I move the date / time into byline I'll remove it! thank you for the code review!

File: src/collective/liveblog/static/styles.css [hunk 2]
```diff
@@ -4,10 +4,6 @@
 }
 .portaltype-liveblog article .field input[type="text"] {
     width: 100%;
-    line-height: 125%;
-    color: black;
-    letter-spacing: -0.05em;
-    margin: inherit -0.05em;
```
Comment (rodfersou) [inline src/collective/liveblog/static/styles.css]: Jigsaw complained for ``margin: inherit -0.05em;`` and I removed some other nonsense stuff (why should we style the input text of the title.. for me looks fine like this)

Comment (hvelarde) [inline src/collective/liveblog/static/styles.css]: ask @agnogueira

**Pull request merged**